lib/common/tests/flags/pcmk_all_flags_set.c

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 the Pacemaker project contributors
+ *
+ * The version control history for this file may have further details.
+ *
+ * This source code is licensed under the GNU Lesser General Public License
+ * version 2.1 or later (LGPLv2.1+) WITHOUT ANY WARRANTY.
+ */
+
+#include <crm_internal.h>
+
+static void
+all_set(void) {
+    g_assert(pcmk_all_flags_set(0x000, 0x003) == false);
+    g_assert(pcmk_all_flags_set(0x00f, 0x003) == true);
+    g_assert(pcmk_all_flags_set(0x00f, 0x010) == false);
+    g_assert(pcmk_all_flags_set(0x00f, 0x011) == false);
+}
```

Comment:
It might be interesting to test `pcmk_all_flags_set(0x000, 0x000)` and `pcmk_all_flag_set(0x00f, 0x000)`.

Reply:
I had thought about that at some point and decided there was no clear interpretation for what should happen when flags are 0, so I just ignored it. Current behavior when flags are 0: "any flags" is always false, "all flags" is always true. I think that's acceptable, so I'll document it and add the unit tests. However I'll do that in another PR since this one is already tested.